CMakeLists.txt

```
@@ -125,7 +125,7 @@ add_subdirectory (src)
 
 
 
-configure_file(${PROJECT_SOURCE_DIR}/WCSimConfig.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/WCSimConfig.cmake @ONLY)
+configure_file(cmake/WCSimConfig.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/WCSimConfig.cmake @ONLY)
```

```suggestion
configure_file(${PROJECT_SOURCE_DIR}/cmake/WCSimConfig.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/WCSimConfig.cmake @ONLY)
```
Would it be better or worse to use the full path, rather than partial path here? (I don't know the answer, you're the CMake expert!)